ampoule/test/test_process.py

```diff
@@ -903,3 +915,22 @@ def _work(_):
         return pp.start(
             ).addCallback(_work
             ).addCallback(lambda _: pp.stop())
+
+    def test_processRestartAfterTimeout(self):
```

Comment:
Can you add a test docstring here that explains what's being tested for?  Given that these tests are live-fire use-an-actual-reactor stuff, it's sometimes hard to get useful information out of a failure message (or more likely... a hang 😢).

(The test suite sets a bad example, and what we *should* have here is something running against a nice in-memory mock, with just a few integration tests to verify that bootstrapping works as expected, but I digress and that certainly isn't your fault...)

Reply:
https://jml.io/test-docstrings/ is a good, and very short, resource on how to write good test docstrings in the Twisted, and hence Ampoule, style

Reply:
Added in 39ddc3f.